content/applications/marketing/surveys/live_session.rst

```diff
@@ -0,0 +1,219 @@
+====================
+Live Session surveys
+====================
+
+With the Odoo *Surveys* application, users can enhance in-person demonstrations and presentations
+with the *Live Session* feature.
+
+*Live Session* surveys function the same way as a normal survey, but with a host or moderator, who
+presents the questions to participants, reveals their responses in real-time, and controls the tempo
+of the survey.
+
+In *Live Session* surveys, participants access the survey experience via a custom URL, and sign in
+with an optional access code. When the survey has begun, the host presents one question at a time.
+
+Then, the audience of participants submit their answer, either via their computer or mobile device,
+and once the resposnes have been gathered, the host reveals all the participant's responses, in
+real time, with each answer's results displayed as a bar graph.
+
+.. image:: live_session/live-session-concept-sample.png
+   :align: center
+   :alt: Rendered concept of how a Live Session question and answer result appears in Odoo Surveys.
+
+Create Live Session survey
+==========================
+
+To create a *Live Session* survey, begin by opening the *Surveys* application. From the
+:guilabel:`Surveys` dashboard, click the :guilabel:`New` button to reveal a blank survey form.
+
+Any of the survey type options (:guilabel:`Survey`, :guilabel:`Live Session`,
+:guilabel:`Assessment`, or :guilabel:`Custom`), presented as radio buttons at the top of the survey,
+can be used to create a *Live Session*.
+
+However, selecting the :guilabel:`Live Session` survey type radio button streamlines the process
+because Odoo auto-selects the optimal settings and options for a *Live Session* survey when that
+option is selected.
+
+.. important::
+   If the :guilabel:`Is a Certification` option is enabled in the :guilabel:`Options` tab, the
+   survey **cannot** be used as a *Live Session* survey.
+
+With the desired survey radio button option selected, proceed to :doc:`create a survey
+<../surveys/create>` with :doc:`questions and sections <../surveys/time_random>`.
+
+While creating questions for the *Live Session* survey, open the :guilabel:`Options` tab on the
+:guilabel:`Open: Sections and Questions` pop-up form, in order to reveal the :guilabel:`Live
```

Comment:
Modal UI element instead of pop-up, and small thing- since were in "create" rather than "edit" the modal title will be "Create: ..." when creating a question :)
```suggestion
:guilabel:`Create: Sections and Questions` modal, in order to reveal the :guilabel:`Live
```

Reply:
in reference to our recent discussion re: "modal v. pop-up TBD", I'm going to keep the pop-up phrasing so as to avoid confusion with the novice readers, but will implement the other aspects of THIS and every other suggestion 👍 Thanks